cli/cmd/stat.go

```diff
@@ -200,8 +203,12 @@ func writeStatsToBuffer(resp *pb.StatSummaryResponse, reqResourceType string, w
 				maxNamespaceLength = len(namespace)
 			}
 
+			meshedCount := fmt.Sprintf("%d/%d", r.MeshedPodCount, r.RunningPodCount)
+			if resourceKey == k8s.Authorities {
```

Comment:
Thanks for adding this! Rather than checking the resource type, I think it might be more future-proof to check `if r.RunningPodCount == 0`.

Reply:
that's fair. I'd originally wanted to keep the `0/0` as a clearer indicator of 0 running pods, as opposed to `-` which I wanted to imply `N/A`. (Like, a 0 value vs a null value) What do you think?

Reply:
Hmm, yeah, I hadn't thought about that. Like, if we scaled a deployment down to 0 replicas? In that case I think you're totally right that 0/0 would be more useful. Totally fine to leave this as is then.